README.md

```diff
@@ -109,7 +109,18 @@ List of external core examples adopted to the latest framework:
 - [COMET-CHAM from Virginia Tech](https://github.com/kammoh/comet_cham_lwc_v2/tree/asic): VHDL (`asic` branch)
 - [SpoC from Virginia Tech](https://github.com/kammoh/comet_cham_lwc_v2/tree/asic): Verilog (`asic` branch)
 
-
+## Popular Configurable LWC Options
+
+### NIST_LWAPI.vhd
```

Comment:
I'm in the process of moving the user configurable setting here to generics in LWC, passed down from LWC_TB and propagated to CryptoCore. It's a WIP and I'm not sure about the implications. Just a heads up.

Reply:
Thats fine. Just update this when you get done.